.github/workflows/pull-requests.yml

```diff
@@ -26,6 +26,16 @@ jobs:
         with:
           command: workflow--check--branch-name
 
+  workflow--check--rebased-on-main:
+    runs-on: [self-hosted, ci]
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - uses: ./.github/actions/make/
+        with:
+          command: workflow--check--rebased-on-main
```

Comment:
Where is this command?

Reply:
Should be able to find it in scripts/workflow and you can find all the github and branch based commands :)